ush/parsing_model_configure_FV3.sh

```diff
@@ -19,29 +19,20 @@ fi
 
 rm -f model_configure
 cat >> model_configure <<EOF
-total_member:            $ENS_NUM
 print_esmf:              ${print_esmf:-.true.}
-PE_MEMBER01:             $NTASKS_TOT
 start_year:              ${tPDY:0:4}
 start_month:             ${tPDY:4:2}
 start_day:               ${tPDY:6:2}
 start_hour:              ${tcyc}
 start_minute:            0
 start_second:            0
-fhrot:                   ${IAU_FHROT:-0}
 nhours_fcst:             $FHMAX
-RUN_CONTINUE:            ${RUN_CONTINUE:-".false."}
-ENS_SPS:                 ${ENS_SPS:-".false."}
+fhrot:                   ${IAU_FHROT:-0}
 
 dt_atmos:                $DELTIM
-output_1st_tstep_rst:    .false.
 calendar:                ${calendar:-'julian'}
-cpl:                     ${cpl:-".false."}
-memuse_verbose:          ${memuse_verbose:-".false."}
-atmos_nthreads:          $NTHREADS_FV3
-use_hyper_thread:        ${hyperthread:-".false."}
-ncores_per_node:         $cores_per_node
 restart_interval:        $restart_interval
```

Comment:
@WalterKolczynski-NOAA Looks like we can just change the line: 
restart_interval: $restart_interval 
to be 
restart_interval: $restart_interval  -1 
and we'll be good to go.  If any updates beyond this need to be done I'd appreciate help because I haven't run aerosols or their restarts and the P7 runs seem fine as is.

Reply:
This is counter to the "not major changes to feature/coupled-crow".  This is also stopping P7 for P7.1 issues which can be handled later, no?  We don't have to have the latest and greatest right now, we just need for it to work, no? How long is it going to take to introduce this? Are we still on the timeline that you said at the meeting this morning or is this delaying things?

Reply:
Is there an error message somewhere showing that the aerosols restart is broken as in this PR? Or without the simple change of adding the -1?